app/ui/archiveTile.js

```diff
@@ -262,11 +262,15 @@ module.exports.wip = function(state, emit) {
         <label
           for="file-upload"
           class="flex flex-row items-center justify-between w-full p-2 cursor-pointer"
-          title="${state.translate('addFilesButton')}"
+          title="${state.translate('addFilesButtonWithSize', {
+            size: bytes(1000000)
```

Comment:
obv this is a placeholder

Reply:
I don't like appending the size here. It makes sense on the home page button though.